app/views/Project/Tagging/Export/ExportHistory/TableActions/styles.css

```diff
@@ -2,6 +2,19 @@
     display: flex;
     align-items: center;
     flex-wrap: wrap;
-    gap: var(--dui-spacing-extra-small);
+    gap: var(--dui-spacing-small);
     justify-content: flex-end;
+
+    .button {
+        padding: var(--tui-spacing-small);
+
+        .children {
+            display: flex;
+            justify-content: center;
+            padding: 0;
+            width: 1.5em;
+            height: 1.5em;
+            font-size: 1.2em;
```

Comment:
Why are we directly using `em` instead of design tokens?

Reply:
@AdityaKhatri